components/playlist/playlist_data_source.cc

```diff
@@ -45,7 +45,7 @@ PlaylistDataSource::PlaylistDataSource(PlaylistService* service)
 PlaylistDataSource::~PlaylistDataSource() = default;
```

Comment:
Let's update the comments in the header!

Reply:
Great! I missed that 😀